lib/ansible/modules/system/timezone.py

```diff
@@ -460,7 +465,9 @@ class SmartOSTimezone(Timezone):
 
     def __init__(self, module):
         super(SmartOSTimezone, self).__init__(module)
-        self.settimezone = self.module.get_bin_path('sm-set-timezone', required=True)
+        self.settimezone = self.module.get_bin_path('sm-set-timezone', required=False)
+        if not self.settimezone:
+            module.fail_json(msg='sm-set-timezone not found. Make sure the smtools package is installed.')
```

Comment:
It's actually in `description`, but with not so many words: `  - On SmartOS the C(sm-set-timezone) utility is used to set the zone timezone.` It's highly unlikely someone has a zone without `smtools` as it comes pre-installed even in the `minimal` images.